x-pack/plugins/monitoring/server/lib/__tests__/helpers.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const response = {
+  hits: {
+    hits: [
+      {
+        _source: {
+          beats_stats: {
+            timestamp: 1515541592880,
+            beat: {
+              uuid: 'fooUuid',
+              host: 'beat-listing.test',
+              name: 'beat-listing.test-0101',
+              type: 'filebeat',
+              version: '6.2.0'
+            },
+            metrics: {
+              beat: {
+                memstats: {
+                  memory_alloc: 2340
+                }
+              },
+              libbeat: {
+                output: {
+                  type: 'Redis',
+                  write: {
+                    bytes: 140000,
+                    errors: 8,
+                  },
+                  read: {
+                    errors: 3,
+                  }
+                },
+                pipeline: {
+                  events: {
+                    total: 23000
+                  }
+                }
+              }
+            }
+          }
+        },
+        inner_hits: {
+          earliest: {
+            hits: {
+              hits: [
+                {
+                  _source: {
+                    beats_stats: {
+                      timestamp: 1515534342000,
+                      metrics: {
+                        libbeat: {
+                          output: {
+                            write: {
+                              bytes: 4000,
+                              errors: 3,
+                            },
+                            read: {
+                              errors: 1,
+                            }
+                          },
+                          pipeline: {
+                            events: {
+                              total: 2300
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    ]
+  }
+};
+
+export const defaultResponseSort = (handleResponse) => {
+  const responseMulti = { hits: { hits: [] } };
+  const hit = response.hits.hits[0];
+  const version = ['6.6.2', '7.0.0-rc1', '6.7.1'];
+
+  for (let i = 0, l = version.length; i < l; ++i) {
```

Comment:
This is a nit-picky suggestion but given that we are only referencing `l` as part of the loop's evaluation, can we drop the declaration of `l` altogether and simply have `for (let i = 0; i < version.length; ++i)`? In the code we currently have, `l` is essentially a constant that is only referenced once, so I don't think we lose any readability by referencing the array's length directly.

Again, just a suggestion, I leave it up to your discretion.

Reply:
I guess since this is a small loop it wouldn't make any difference. But, there is actually a performance gain doing it this way since `Array.length` is only calculated once (and not per every iteration).

I'll keep it this way for now since this is just a unit test, but I'll be sure to favor readability in the future

Reply:
That's fine! 👍